src/app/components/RuntimeEvents/RuntimeEventDetails.tsx

```diff
@@ -264,13 +264,17 @@ export const RuntimeEventDetails: FC<{
               />
               {addressSwitchOption === AddressSwitchOption.Oasis && <CopyToClipboard value={event.body.to} />}
             </dd>
-            <dt>{t('runtimeEvent.fields.amount')}</dt>
-            <dd>
-              {t('common.valueInToken', {
-                ...getPreciseNumberFormat(event.body.amount.Amount),
-                ticker: event.body.amount.Denomination,
-              })}
-            </dd>
+            {event.body.amount?.Amount && event.body.amount?.Denomination && (
```

Comment:
I'd add TODO. This is only needed because some TODO events temporarily abuse this for rendering.

Or, just:
```tsx
    case RuntimeEventType.consensus_accountsdelegate: // TODO show this properly
    case RuntimeEventType.consensus_accountsundelegate_start: // TODO show this properly
    case RuntimeEventType.consensus_accountsundelegate_done: // TODO show this properly
      return (
        <div>
          <b>{event.type}</b>
          <br />
          <pre>{JSON.stringify(event, null, ' ')}</pre>
        </div>
      )
    default:
```